service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsSubmittedControllerTest.java

```diff
@@ -65,7 +68,7 @@ void setup() {
 
     @Test
     void submittedEventShouldNotifyHmctsAdminWhenCtscToggleIsDisabled() throws Exception {
-        postSubmittedEvent(enableSendToCtscOnCaseDetails(NO));
+        postSubmittedEvent(enableSendToCtscOnCaseDetails(NO, YES));
```

Comment:
I think we should rename `enableSendToCtscOnCaseDetails` method to something more generic, like `buildCaseDetails` or `caseDetailsWithSendToCtscAndPbaPayment` to make it more readable